spec.bs

```diff
@@ -185,44 +185,59 @@ Each {{URLPattern}} object has an associated <dfn for=URLPattern>search componen
 Each {{URLPattern}} object has an associated <dfn for=URLPattern>hash component</dfn>, a [=component=], which must be set upon creation.
 
 <dl class="domintro non-normative">
-  <dt><code>{{URLPattern}} . {{URLPattern/protocol}}</code>
+  <dt><code>|urlPattern| = new {{URLPattern/constructor(input, baseURL)|URLPattern}}(|input|, |baseURL|)</code></dt>
+  <dd class=allow-2119>
```

Comment:
Turning off the 2119 checking is a smell. I'd suggest just saying "can" instead of "may".

Reply:
I have been doing that for most of my notes.  I haven't been able to figure out consistently what terms I'm using are triggering it.  Honestly the checks are annoying and don't seem to add value to me.

Reply:
I started adding the exception after many iterations trying to change my phrasing and gave up.

Reply:
The words are the words in https://infra.spec.whatwg.org/#conformance . It is pretty confusing for me as a reader to encounter those words in non-normative contexts, so I think it's worth trying to avoid bypassing the check when possible.

Reply:
The problem I run into is when I'm trying to describe things the web developer must do or we as the spec writer must algorithmically do, but they are not MUST statements directed at the implementor.  How do I phrase that?

And is it that confusing if you didn't notice the other dozen or so exceptions I already have in the doc?

Reply:
You use the alternatives listed in https://infra.spec.whatwg.org/#conformance:

> For non-normative content "strongly encouraged", "strongly discouraged", "encouraged", "discouraged", "can", "cannot", "could", "could not", "might", and "might not" can be used instead. 

> And is it that confusing if you didn't notice the other dozen or so exceptions I already have in the doc?

This is indeed the first time it's really struck me, besides the last PR where allow-2119 seemed legitimate since it was a non-normative note explicitly discussing a lack of "required" normative requirements.

But it seems also pretty easy to fix in other places. E.g. "We found a protocol suffix, so this must be an absolute URLPattern constructor string" -> "We found a protocol suffix, so this **is** an absolute URLPattern constructor string".